dev-app/src/screens/CollectCardPaymentScreen.tsx

```diff
@@ -725,6 +727,16 @@ export default function CollectCardPaymentScreen() {
               />
             }
           />
+          <ListItem
+            title="Request DCC (requires Update PaymentIntent)"
```

Comment:
what happens if this is set while `updatePaymentIntent` is false?

Reply:
alternatively you can make this visible / editable only if Update PaymenIntent is checked

Reply:
> what happens if this is set while updatePaymentIntent is false?
we return the collect call with a failure (telling users that update PI must be enabled)

> alternatively you can make this visible / editable only if Update PaymenIntent is checked
yup, i'll disable it until update PI enabled